tests/system/Database/Live/ForgeTest.php

```diff
@@ -472,28 +481,16 @@ public function testForeignKey()
         $this->forge->addForeignKey('users_id', 'forge_test_users', 'id', 'CASCADE', 'CASCADE');
 
         $tableName = 'forge_test_invoices';
-        if ($this->db->DBDriver === 'OCI8') {
-            $tableName = 'forge_test_inv';
-        }
 
         $this->forge->createTable($tableName, true, $attributes);
 
         $foreignKeyData = $this->db->getForeignKeyData($tableName);
 
```

Comment:
Now that everything is standardized with constraint names we don't need all the different implementations.